frontend/src/concepts/pipelines/content/tables/experiment/ExperimentTableRow.tsx

```diff
@@ -23,12 +23,20 @@ const ExperimentTableRow: React.FC<ExperimentTableRowProps> = ({
   actionColumnItems,
 }) => {
   const { namespace } = usePipelinesAPI();
+  const location = useLocation();
+
+  const isArchived = location.pathname.includes('/archived');
```

Comment:
cant you just check if the experiment is archived or not instead of the route itself?

like 

experiment.storage_state === StorageStateKF.ARCHIVED